src/scss/_admonitions.scss

```diff
@@ -1,14 +1,20 @@
-// Admonitions CSS inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
+// Admonitions CSS originally inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
+
+@function hexToRGB($hex) {
+  @return red($hex), green($hex), blue($hex);
```

Comment:
Where is this function being used? (or a leftover?)

Reply:
Hm, this might indeed be a leftover.